dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/ops/SharingContext.kt

```diff
@@ -9,21 +9,35 @@ class SharingContext<Initial, Return> {
     internal var areSameType = true
 
     /**
-     * Express the value [toReturn] after the [share] computation is done.
+     * Express the value [toReturn] when the [share] computation is done.
+     * Usually the [toReturn] value is different from the [toSend],
+     * otherwise use [share] without calling this function to return the value [toSend].
      * It can be used with checks after the invocation.
-     *
      * ## Example
      * ```
-     * val res3 = share(0) {f ->
-     *   val minValue = f.toMap().minBy { it.value }.value
-     *   minValue butReturn if (min > 1) "Hello" else null
+     * share(0) {
+     *   sendButReturn(it.toMap().maxBy { v -> v.value }.value, "A string")
      * }
      * ```
      */
```

Comment:
```suggestion
     * ```
     * The invoke of [sendButReturn] as the last statement of the body of the [share],
     *  sent to the neighbors the [toSend] value, but returns from the [share] the [toReturn] value.
     */
```